x/evm/keeper/msg_server.go

```diff
@@ -1455,13 +1457,17 @@ func (s msgServer) SignCommands(c context.Context, req *types.SignCommandsReques
 		return nil, err
 	}
 
+	commandList := types.CommandIDsToStrings(commandBatch.GetCommandIDs())
+	s.Logger(ctx).Debug(fmt.Sprintf("signing batched command %s with commands %v", batchedCommandsIDHex, commandList))
```

Comment:
I suggest loop through `types.CommandIDsToStrings(commandBatch.GetCommandIDs())` and log command IDs each with its own entry. Currently, it may result in a huge line of logging and hurt readability.